docs/machine-learning/ml-data-exploration.md

```diff
@@ -0,0 +1,41 @@
+# Data Exploration
+
+After [envisioning](./ml-problem-formulation-envisioning.md), the next step is to confirm resource access and then dive deep into the available data through data exploration workshops.
+
+## Accessing Resources
+
+Prior to diving into data exploration workshops, it is important to confirm that you have access to the necessary resources (including data).
+
+Below is an **example** list of questions to consider before starting a data exploration workshop.
+
+1. What are the requirements for an account to be set up in order for the team to access data and compute resources?
+2. Are there security requirements around accessing resources (Subscriptions, Azure Resources, project management, etc.) such as VPN, 2FA, jump boxes, etc.?
+3. Data access:
+    * Is it on-prem or on Azure already?
+    * If it is on-prem, can we move the needed data to Azure under the appropriate subscription? Who has permission to move the data?
+    * Is the data access approved from a legal/compliance perspective?
+4. Computation:
+    * Is a VPN needed for the project team to access these computation nodes (Virtual Machines, Databricks clusters, etc) from their work PCs/Macs?
```

Comment:
nit: Not sure about his 100%, but I believe that for all Azure resources (VMs, Databricks), a VPN is not required. A VPN might be required for accessing on-prem resources like databases or compute.

Reply:
I have had experiences where I needed to connect to the VPN to access the contents of the resources. I was able to access the page itself (e.g., Azure Machine Learning Workspace), but non of the relevant content would display.

In light of that, do you feel it is okay for us to leave this as-is?